src/services/cookies.js

```diff
@@ -0,0 +1,11 @@
+export default {
```

Comment:
I have nothing against cookies, but @AnthonySendra leveraged the SessionStorage in the Klack App, wouldn't it be better? Or even LocalStorage if we want real persistence...
As far as I understand from [this thread](http://stackoverflow.com/questions/3220660/local-storage-vs-cookies), Cookies are essentially useful when their content must be sent to the server at every request (which is not our case).
That being said, this module is useful and should not be dropped even if we choose another method to implement the session.